### PR TITLE
Add commonly requested documents

### PR DESCRIPTION
**Pull request by jayroh**
Change verbage and content around the documents page to reflect what
exists currently at GetCalFresh.

Trello:

* https://trello.com/c/PMl961LU/70-when-i-am-prompted-with-the-option-to-upload-documents-i-want-to-receive-helpful-guidance-about-what-is-actually-required-and-he
* https://trello.com/c/eKV6U8jR/71-as-a-client-i-want-guidance-about-which-documents-to-upload-when-prompted

### CODE REVIEW

**Comment by jessieay** [comment]
2 requests:

- screenshot
- link to trello card

🤗

File: app/views/documents/edit.html.erb [hunk 1]
```diff
@@ -27,13 +38,23 @@
         </script>
       </div>
 
-      <div class="form-card__target" id="document-drop">
-        <i class="icon-file_upload"></i>
-        Click, Tap, Or Drag &amp; Drop to Upload Photos/Documents
+      <div class="form-card__documents__ctas">
+        <button type="button" class="button button--next button--cta button--full-width" id="click-to-upload">
+          <i class="icon-file_upload"></i>
+          Add docs
```
Comment (jessieay) [inline app/views/documents/edit.html.erb]: "Add documents" ?

File: app/views/shared/_commonly_requested_documents.html.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  <div class="media-box__content">
+    <p><b>Proof of all money</b> you received in the last 30 days, like pay stubs from your employer or bank statements.</p>
+  </div>
+</div>
+<div class="media-box with-padding-small">
+  <div class="media-box__media">
+    <div class="emoji emoji--med emoji--house"></div>
+  </div>
+  <div class="media-box__content">
+    <p><b>Proof of where you live.</b> Any document that links your name to your address like your utility bill.</p>
```
Comment (jessieay) [inline app/views/shared/_commonly_requested_documents.html.erb]: is `b` better than `strong` these days? I can never keep track

File: app/views/shared/_commonly_requested_documents.html.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  </div>
+</div>
+<div class="media-box with-padding-small">
+  <div class="media-box__media">
+    <div class="emoji emoji--med emoji--dollar-sign"></div>
+  </div>
+  <div class="media-box__content">
+    <p><b>Proof of expenses</b>, like rent, utilities, and certain medical bills (the more expenses you show the more CalFresh you can get).</p>
```
Comment (jessieay) [inline app/views/shared/_commonly_requested_documents.html.erb]: I 👁  CalFresh (sub for MichiganBenefits or FAP? )

File: app/views/shared/_commonly_requested_documents.html.erb [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  </div>
+</div>
+<div class="media-box with-padding-small">
+  <div class="media-box__media">
+    <div class="emoji emoji--med emoji--memo"></div>
+  </div>
+  <div class="media-box__content">
+    <p><b>Self attestations</b>. If you don't have an official document, you can write a statement on paper saying how much money you received, and sign and date it.</p>
```
Comment (jessieay) [inline app/views/shared/_commonly_requested_documents.html.erb]: break line for readability?

**Comment by jayroh** [comment]
> 2 requests:

D'OH. I forget these too often

**Comment by jessieay** [comment]
@jayroh cool -- also, we are currently using the "Trello github powerup" to connect PRs and tickets. Happy to discuss something else, but I find it kind of nifty since it reports back to Trello on the status of the PR.

File: app/views/shared/_commonly_requested_documents.html.erb (hunk 2 above)
Comment (jayroh) [inline app/views/shared/_commonly_requested_documents.html.erb]: This file is entirely copy-pasta'd from GCF :). So I trust their (Norris?) semantic html intuition!

File: app/views/shared/_commonly_requested_documents.html.erb (hunk 3 above)
Comment (jayroh) [inline app/views/shared/_commonly_requested_documents.html.erb]: Whoopsie!  Egg => face.

**Comment by jayroh** [comment]
[As a client, after I uploading docs or choosing not to, I want to see a confirmation page telling me I am FINISHED! and giving guidance on what to do next](https://trello.com/c/DVStptea/82-as-a-client-after-i-uploading-docs-or-choosing-not-to-i-want-to-see-a-confirmation-page-telling-me-i-am-finished-and-giving-guid)

**Pull request merged**